pepper-apis/src/main/java/org/broadinstitute/ddp/json/mailinglist/GetMailingListResponse.java

```diff
@@ -16,13 +16,16 @@
     private String info;
     @SerializedName("dateCreated")
     private long dateCreated;
+    @SerializedName("isoLanguageCode")
```

Comment:
@SimoneMaiwald : Simone adding "isoLanguageCode" to MailingList Response as part of adding platform level  language support for MailingList